src/Blocks/BlockCrops.h

```diff
@@ -81,9 +43,51 @@ class cBlockCropsHandler :
 					break;
 				}
 			}
+			return;
 		}
+
+		// Fully grown, drop the produce:
```

Comment:
I think `produce` is the correct word, it does mean "fruit, vegetables etc. that have grown and are collected"; `product` is too generic. Of course I could be wrong, any English natives care to chime in? @bearbin @tigerw ?